packages/camelot/src/Camelot.test.ts

```diff
@@ -1,210 +1,40 @@
-                            },
-                          ],
-                        },
-                      },
-                    ],
-                    amountIn: {
-                      $gte: '1000000',
-                    },
-                    amountOutMinimum: {
-                      $gte: '500000000000000',
-                    },
-                  },
-                },
-              ],
-            },
-            {
-              $abi: PARASWAP_ABI,
-              $or: [
-                {
-                  data: {
-                    fromToken: Tokens.USDT,
-                    fromAmount: {
-                      $gte: '1000000',
-                    },
-                    toAmount: {
-                      $gte: '500000000000000',
-                    },
-                    toToken: Tokens.WETH,
-                    partner: '0x353D2d14Bb674892910685520Ac040f560CcBC06',
-                  },
-                },
-                {
-                  data: {
-                    fromToken: Tokens.USDT,
-                    fromAmount: {
-                      $gte: '1000000',
-                    },
-                    toAmount: {
-                      $gte: '500000000000000',
-                    },
-                    path: {
-                      $last: {
-                        to: Tokens.WETH,
-                      },
-                    },
-                    partner: '0x353D2d14Bb674892910685520Ac040f560CcBC06',
-                  },
-                },
-                {
-                  data: {
-                    fromToken: Tokens.USDT,
-                    fromAmount: {
-                      $gte: '1000000',
-                    },
-                    toAmount: {
-                      $gte: '500000000000000',
-                    },
-                    path: {
-                      $last: {
-                        path: {
-                          $last: {
-                            to: Tokens.WETH,
-                          },
-                        },
-                      },
-                    },
-                    partner: '0x353D2d14Bb674892910685520Ac040f560CcBC06',
-                  },
-                },
-                {
-                  data: {
-                    assets: {
-                      $and: [
-                        {
-                          $first: Tokens.USDT,
-                        },
-                        {
-                          $last: Tokens.WETH,
-                        },
-                      ],
-                    },
-                    fromAmount: {
-                      $gte: '1000000',
-                    },
-                    toAmount: {
-                      $gte: '500000000000000',
-                    },
-                    partner: '0x353D2d14Bb674892910685520Ac040f560CcBC06',
-                  },
-                },
-              ],
-            },
-          ],
-        },
+        chainId: 1,
```

Comment:
Don't think mainnet is the right network for Camelot

Reply:
This test checks that the format of the action filter is correct based on the inputs. 

Its a standard test I started including in all new plugins.

It checks:

  - That the filter is an object
  - It has a chainId property (or sourceChainId if bridge)
  - The chainId matches the input chainId
  - It has a "to" property of type Address or an object with an $or property as the only key
  - It has a valid abi operator